examples/httpDeviceShifu/deployment/http-deviceshifu-telemetryservice.yaml

```diff
@@ -5,18 +5,12 @@ metadata:
   name: push-endpoint-1
   namespace: devices
 spec:
-  type: HTTP
-  address: http://192.168.15.48:12345/endpoint1
+  telemetrySeriveEndpoint: http://telemetryservice.shifu-service.svc.cluster.local
   serviceSettings:
-    HTTPSetting:
-      username: admin
-      password: password
---- #telemetry_service2.yaml
-apiVersion: shifu.edgenesis.io/v1alpha1
-kind: TelemetryService
-metadata:
-  name: push-endpoint-2
-  namespace: devices
-spec:
-  type: HTTP
-  address: http://192.168.15.48:12345/api1
+    SQLSetting:
+      serverAddress: 192.168.14.163:6041
+      username: root
+      secret: taosdata
```

Comment:
Maybe we should encode the secret with k8s secret and get it with ENV_VAR, and here should be the name of the ENV_VAR, and add a comment to state it.

Reply:
handle it after shifuctl implememt